pkg/admission/pluginconfig_webhook.go

```diff
@@ -178,15 +184,18 @@ func validatePluginConfigOptionValues(pluginConfig *greenhousev1alpha1.PluginCon
 	return apierrors.NewInvalid(pluginConfig.GroupVersionKind().GroupKind(), pluginConfig.Name, allErrs)
 }
 
-func validatePluginConfigClusterExists(ctx context.Context, pluginConfig *greenhousev1alpha1.PluginConfig, c client.Client) error {
-	// TODO: Enforce clusterName on Plugins with backend.
-	//  We allow PluginConfigs without cluster reference during migration.
-	//  Later a PluginConfig with a backend must have a clusterName configured. Frontend-only plugins are allowed without a clusterName.
```

Comment:
This will break for the heureka plugin, which is frontend-only.

Reply:
i excluded fontend-only plugins from the list